src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -129,4 +137,48 @@ public function getStatusCode()
     {
         return $this->statusCode;
     }
+
+    /**
+     * Sets the faked user inputs.
+     *
+     * @param array $inputs
+     *
+     * @return CommandTester
+     */
+    public function setUserInputs(array $inputs)
+    {
+        $this->userInputs = $inputs;
+
+        return $this;
+    }
+
+    /**
+     * Gets the faked user inputs.
+     *
+     * @return array $inputs
+     */
+    public function getUserInputs()
+    {
+        return $this->userInputs;
+    }
+
+    /**
+     * Creates the command input stream.
+     */
+    private function createInputStream()
+    {
+        $stream = fopen('php://memory', 'r+', false);
+
+        fputs($stream, implode("\n", $this->userInputs));
```

Comment:
I tried to follow this http://symfony.com/doc/current/components/console/helpers/questionhelper.html#testing-a-command-that-expects-input as possible, please let me know if you still think that I should use fwrite.